x-pack/plugins/infra/public/components/logging/log_source_error_page.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiEmptyPrompt,
+  EuiPageTemplate,
+  EuiSpacer,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+import { SavedObjectNotFound } from '../../../../../../src/plugins/kibana_utils/common';
+import {
+  FetchLogSourceConfigurationError,
+  FetchLogSourceStatusError,
+  ResolveLogSourceConfigurationError,
+} from '../../../common/log_sources';
+import { useLinkProps } from '../../hooks/use_link_props';
+
+export const LogSourceErrorPage: React.FC<{
```

Comment:
This is a fork of the previously shared source error page, because the log source handling has diverged too much from the metrics source handling.